### PR TITLE
Realtime gateway cache invalidation for revoked tokens

### PR DESCRIPTION
**Pull request by bhathiya**

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/conf_templates/templates/repository/conf/api-manager.xml.j2 [hunk 1]
```diff
@@ -767,7 +767,6 @@
         </BlockCondition>
         <JMSConnectionDetails>
             <Enabled>{{apim.throttling.enable_decision_connection}}</Enabled>
-            <Destination>{{apim.throttling.jms.destination}}</Destination>
```
Comment (ruks) [inline features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/conf_templates/templates/repository/conf/api-manager.xml.j2]: We can remove default config value 'apim.throttling.jms.destination' from here[1]

https://github.com/wso2/carbon-apimgt/blob/master/features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/conf_templates/org.wso2.carbon.apimgt.core.default.json#L88

Comment (bhathiya) [reply features/apimgt/org.wso2.carbon.apimgt.core.feature/src/main/resources/conf_templates/templates/repository/conf/api-manager.xml.j2]: Fixed in https://github.com/bhathiya/carbon-apimgt/commit/725c37293a88cf5d574c6eaedae36fd906fef75f

File: components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/Utils.java [hunk 2]
```diff
@@ -19,7 +19,16 @@
 import org.apache.axiom.om.OMAbstractFactory;
 import org.apache.axiom.om.OMDocument;
 import org.apache.axiom.om.OMElement;
-import org.apache.axiom.soap.*;
+import org.apache.axiom.soap.SOAPEnvelope;
```
Comment (harsha89) [inline components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/Utils.java]: Are these imports used?

Comment (bhathiya) [reply components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/Utils.java]: yes, but not for this feature.

**Comment by bhathiya** [comment]
@shaniR @Mariangela With this feature, we can remove immediate token revocation information from the hazelcast clustering docs. We don't need hazelcast for immediate revocation of tokens among the gateways, anymore.

**Pull request merged**